activerecord/test/cases/adapters/postgresql/enum_test.rb

```diff
@@ -31,7 +31,7 @@ def setup
 
   teardown do
     @connection.drop_table "postgresql_enums", if_exists: true
-    @connection.execute "DROP TYPE IF EXISTS mood"
+    @connection.drop_enum "mood"
```

Comment:
What do you think about adding `if_exists: true` support option for drop_enum?

Reply:
Oh yeah - good idea 👍